src/test/resources/alma-fix/990133067580206441.json

```diff
@@ -79,8 +79,8 @@
     "id" : "http://ld.zdb-services.de/resource/2163340-X",
     "label" : "ZDB-Ressource"
   }, {
-    "id" : "https://d-nb.info/026537966",
-    "label" : "DNB-Ressource"
+    "id" : "http://dbis.uni-regensburg.de/frontdoor.php?titel_id=1815",
+    "label" : "DBIS-Ressource"
   }, {
```

Comment:
In https://github.com/hbz/lobid-resources/issues/1864#issuecomment-2296603227 it's said to _add_ a link to DBIS, not to drop the link to dnb.

Reply:
This is weird